src/p2p/net_node.inl

```diff
@@ -2390,10 +2393,13 @@ namespace nodetool
               LOG_INFO_CC(context, "invalid connection [COMMAND_BROADCAST]");
               return true;
           }
+          // saving connection context info to temp variable as it might be destroyed in case
+          // 'invoke_notify_to_peer' fails - this way LOG_ERROR_CC with context might cause segfault
+          std::string conn_info = epee::net_utils::print_connection_context_short(context);
           if (invoke_notify_to_peer(COMMAND_BROADCAST::ID, blob, context)) {
               announced_peers.insert(context.peer_id);
           } else {
-              LOG_ERROR_CC(context, "failed to invoke COMMAND_BROADCAST");
+              LOG_ERROR("[" << conn_info << "] failed to invoke COMMAND_ANNOUNCE");
```

Comment:
Why COMMAND_ANNONCE, this is `do_broadcast()` method?

Also, if it may produce segfault, why we don't remove it?

Reply:
> Why COMMAND_ANNONCE, this is do_broadcast() method?

my bad, fixed now

> Also, if it may produce segfault, why we don't remove it?

to keep log consistent:
we print 
` MINFO("sending COMMAND_BROADCAST to " << context.peer_id);`
and then, in case it failed we print:
` LOG_ERROR("[" << conn_info << "] failed to invoke COMMAND_BROADCAST");`